src/GraphQL/Mutation/InputHandler/FieldType/RichText.php

```diff
@@ -6,11 +6,11 @@
  */
 namespace EzSystems\EzPlatformGraphQL\GraphQL\Mutation\InputHandler\FieldType;
 
-use eZ\Publish\Core\FieldType\RichText as RichTextFieldType;
 use eZ\Publish\SPI\FieldType\Value;
 use EzSystems\EzPlatformGraphQL\Exception\UnsupportedFieldInputFormatException;
 use EzSystems\EzPlatformGraphQL\GraphQL\Mutation\InputHandler\FieldType\RichText\RichTextInputConverter;
 use EzSystems\EzPlatformGraphQL\GraphQL\Mutation\InputHandler\FieldTypeInputHandler;
+use EzSystems\EzPlatformRichText\eZ\RichText as RichTextFieldType;
```

Comment:
```suggestion
use EzSystems\EzPlatformRichText\eZ\FieldType\RichText as RichTextFieldType;
```
As in PHPDoc below, file in ezplatform-richtext package: https://github.com/ezsystems/ezplatform-richtext/blob/master/src/lib/eZ/FieldType/RichText/Value.php)

Otherwise I'm getting an exception:
```
      "debugMessage": "Class 'EzSystems\\EzPlatformRichText\\eZ\\RichText\\Value' not found",
```
when running a mutation:
```
mutation {
  createFolder(
      input: {
      name: "Test"
      shortName:"Test2"
        shortDescription: {
          format: html
          input: "test"
        }
        description: {
          format: html
          input: "test"
        }
      }
    language: eng_GB
    parentLocationId: 2
  )
  {
    id
    name
  }
}
```